edge-hub/core/src/Microsoft.Azure.Devices.Edge.Hub.Core/config/BrokerConfigValidator.cs

```diff
@@ -58,7 +58,55 @@ public virtual IList<string> ValidateAuthorizationConfig(AuthorizationProperties
             return errors;
         }
 
-        private static void ValidateRule(AuthorizationProperties.Rule rule, int order, List<string> errors)
+        /// <summary>
+        /// Important!: Validation logic should be in sync with validation logic in the Broker.
+        ///
+        /// Validates bridge config and returns a list of errors (if any).
+        /// </summary>
+        public virtual IList<string> ValidateBridgeConfig(BridgeConfig properties)
+        {
+            Preconditions.CheckNotNull(properties, nameof(properties));
+
+            var order = 0;
+            var errors = new List<string>();
+            foreach (var bridge in properties)
```

Comment:
There are multiple bridges in the properties? Is it the whole hierarchy of bridges? How does this work?